### ISSUE TITLE
Selected Files Should Retain Git Status Coloring

### ISSUE THREAD

**Issue by ChrisBaker97** [tclem/atom-solarized-dark-ui#2]
When files are actively selected in the tree view, their Git status coloring is overridden by the highlight color. It would be nice if the file names retained the appropriate Git status coloring.

**Comment by tclem** [comment]
I think this should be fixed on master now. Let me know if it isn't and we can re-open!

**Comment by ChrisBaker97** [comment]
Hi Tim,

I think this is still broken. The coloring works when another file is highlighted in the tree-view sidebar:

![image](https://cloud.githubusercontent.com/assets/2942518/6735798/15c4b9a8-ce37-11e4-9251-4680d390290d.png)

...but when the changed file is selected, the file name still appears white:

![image](https://cloud.githubusercontent.com/assets/2942518/6735777/f9d6f990-ce36-11e4-826d-eff17860ff42.png)

FWIW, this is also a problem in the default atom dark theme:

![image](https://cloud.githubusercontent.com/assets/2942518/6735832/50ea59fc-ce37-11e4-97a5-6e418eb51baf.png)

![image](https://cloud.githubusercontent.com/assets/2942518/6735823/3dddef40-ce37-11e4-9143-d3b6734f0071.png)

However, there is a difference between the two. When the focus is returned to the editor window, the default atom dark theme restores the coloring:

![image](https://cloud.githubusercontent.com/assets/2942518/6735921/e94f5a62-ce37-11e4-9959-85f869950809.png)

The solarized theme doesn't change when the editor gains focus.

So maybe this goes deeper than the solarized themes? Let me know if you think I should submit this as a bug elsewhere instead.

Thanks.

**Comment by ChrisBaker97** [comment]
However, there is a difference between the two after all. When the focus is returned to the editor window, the default atom dark theme restores the coloring:

![image](https://cloud.githubusercontent.com/assets/2942518/6735921/e94f5a62-ce37-11e4-9959-85f869950809.png)

The solarized theme doesn't change when the editor gains focus.

**Comment by tclem** [comment]
Hi @ChrisBaker97!

I'm having trouble seeing this myself. Are you tracking the latest release? What platform are you on (Mac, Window, Linux)?

Here's what I see. I don't get those blue highlights, so I'm wondering if you've got your own styles or another package getting in the way?

Here's what I see with the tree view focused:

![image](https://cloud.githubusercontent.com/assets/136521/6746179/41a5f380-ce81-11e4-91f6-5d1fb69f5250.png)

And here's with the editor focused:

![image](https://cloud.githubusercontent.com/assets/136521/6746192/6a6ee9de-ce81-11e4-8221-d7aeb7a6c86a.png)

Highlighting difference is subtle, but that's on purpose.

**Comment by ChrisBaker97** [comment]
Well, this is awkward... Apparently, at some point, I switched over to the [`solarized-dark-ui`](https://atom.io/themes/solarized-dark-ui) theme from your [`atom-solarized-dark-ui`](https://atom.io/themes/atom-solarized-dark-ui), and that was the one still active. So...once I actually went back to `atom-solarized-dark-ui`, yup, it works great now! Sorry!

(Also, the glaring blue highlighting was an example of the stock [`atom-dark-ui`](https://atom.io/themes/atom-dark-ui) theme, not the solarized themes.)

I'll show myself out now.
